MSAL/src/public/MSALPublicClientApplication.h

```diff
@@ -54,34 +56,33 @@
     templates representing well-formed authorities. It is useful when the authority is obtained at
     run time to prevent MSAL from displaying authentication prompts from malicious pages.
  */
-@property BOOL validateAuthority DEPRECATED_MSG_ATTRIBUTE("MSALPublicClientApplicationConfig.knownAuthorities instead");
+@property BOOL validateAuthority DEPRECATED_MSG_ATTRIBUTE("Use knowAuthorities in MSALPublicClientApplicationConfig instead (create your config and pass it to -initWithConfiguration:error:)");
 
 /*! The authority the application will use to obtain tokens */
-@property (readonly, nonnull) MSALAuthority *authority;
+@property (readonly, nonnull) MSALAuthority *authority DEPRECATED_MSG_ATTRIBUTE("Use authority in MSALPublicClientApplicationConfig instead (create your config and pass it to -initWithConfiguration:error:)");
 
 /*! The client ID of the application, this should come from the app developer portal. */
-@property (readonly, nonnull) NSString *clientId;
+@property (readonly, nonnull) NSString *clientId DEPRECATED_MSG_ATTRIBUTE("Use clientId in MSALPublicClientApplicationConfig instead (create your config and pass it to -initWithConfiguration:error:)");
 
 /*! The redirect URI of the application */
-@property (readonly, nonnull) MSALRedirectUri *redirectUri;
+@property (readonly, nonnull) MSALRedirectUri *redirectUri DEPRECATED_MSG_ATTRIBUTE("Use redirectUri in MSALPublicClientApplicationConfig instead (create your config and pass it to -initWithConfiguration:error:)");
```

Comment:
those are readonly, so maybe remove the note about "(create your config and pass it to -initWithConfiguration:error:)" for readonly ones?

Reply:
done